pkgs/os-specific/linux/vendor-reset/default.nix

```diff
@@ -0,0 +1,35 @@
+
+{ stdenv, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  name = "vendor-reset-${version}-${kernel.version}";
+  version = "0.0.17";
+
+  src = fetchFromGitHub {
+    owner = "gnif";
+    repo = "vendor-reset";
+    rev = "69ec73efd37cf1e28684747fd4e20598b366b8a4";
+    sha256 = "1gakp4clg8zij0kwrp3kp6h4z9vl384xf82k8kkr5zrz28ybgnhy";
+  };
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  hardeningDisable = [ "pic" ];
+
+  makeFlags = [
+    "KVER=${kernel.modDirVersion}"
+    "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+  ];
+
+  installPhase = ''
+    install -D vendor-reset.ko -t "$out/lib/modules/${kernel.modDirVersion}/kernel/drivers/misc/"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Linux kernel vendor specific hardware reset module";
+    homepage = "https://github.com/gnif/vendor-reset";
+    license = licenses.gpl2;
```

Comment:
Only or Plus?

Reply:
Plain GPLv2: https://github.com/gnif/vendor-reset/blob/master/LICENSE

Reply:
That would be gpl2Only.

Reply:
Ok, I'll change it. Now I see that gpl2 is a deprecated alias for gpl2Only.